accounting/serializers.py

```diff
@@ -16,4 +20,28 @@ class Meta:
             'channel_name',
             'date_created',
             'date_updated'
-        )
\ No newline at end of file
+        )
+
+class ProjectDetailSerializer(serializers.ModelSerializer):
```

Comment:
what is the difference between this serializer class and `ProjectSerializer`?

Reply:
i think the right approach to this is to create a separate endpoint to get the pending issues of the project.

something like: `api/projects/issues/pending/`